nucypher/cli/commands/worklock.py

```diff
@@ -194,17 +202,22 @@ def bid(general_config, worklock_options, force, hw_wallet, value):
     value = int(Web3.toWei(Decimal(value), 'ether'))
 
     if not force:
-        paint_bidding_notice(emitter=emitter, bidder=bidder)
-        click.confirm(f"Place WorkLock bid of {prettify_eth_amount(value)}?", abort=True)
+        if not existing_bid_amount:
+            paint_bidding_notice(emitter=emitter, bidder=bidder)
+            click.confirm(f"Place WorkLock bid of {prettify_eth_amount(value)}?", abort=True)
```

Comment:
Should the text here and in other places in the file belong in `nucypher.cli.literature`?

Reply:
Yeah, I think that's OK, but I tried to keep the scope of this reduced. There are some other strings in this module anyway, so it needs a revision some time.